src/component/Inputcity.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import Time from './Time'
+import Weatherrr from './Weatherrr'
+const weather = require('openweather-apis');
+
+function Inputcity() {
+    const [cooord, setcooord] = useState({ lati: '51.5085', long: '-0.1257', cityy: 'London' })
```

Comment:
writing it like `cooord` is so confusing
try to call it something like weatherCoord